### ISSUE TITLE
Update editorial workflow to include creating proposed lesson ticket

### ISSUE THREAD

**Issue by jerielizabeth** [programminghistorian/jekyll#416]
Continuation of #339. 

Update the following files to include new step to editorial workflow to create proposed lesson ticket.
- [x] http://programminghistorian.org/new-lesson-workflow
- [x] http://programminghistorian.org/editor-guidelines

Suggested text for the ticket:

> The Programming Historian has received the following proposal for a lesson on 'PROVISIONAL LESSON TITLE' by AUTHOR(S) NAME(S). This purpose of the lesson is to:
>
> - key learning outcome 1
> - key learning outcome 2
> - key learning outcome 3 (add as needed)
>
> In order to promote speedy publication of this important topic, we have agreed to a submission date of no later than [90 DAYS BY DEFAULT BY LONGER IF AGREED WITH EDITOR]. The author(s) agree to contact the editor in advance if they need to revise the deadline.
>
> If the lesson is not submitted within the AGREED PERIOD, the editor will attempt to contact the authors. If they do not receive an update, this ticket will be closed. The ticket can be reopened at a future date at the request of the author.
>
>The main editorial contact for this lesson is [EDITOR USERNAME].  If there are any concerns from the authors they can contact the Ombudsperson @ianmilligan1 or @amandavisconti.

**Comment by jerielizabeth** [comment]
Question about the default issue text in Github -- is it possible to have two templates that the ticket creator could choose between? @wcaleb @mdlincoln

**Comment by mdlincoln** [comment]
Not from what I can see in the issue documentation :frowning_face: 

A workaround would be to have both texts in the template, and the submitter just needs to delete the unneeded section. Kind of a bother, but it's a simple fix.

**Comment by jerielizabeth** [comment]
@mdlincoln I had a feeling that would be the case. 

I would be ok with having both in the template ... especially since we're supposed to be doing this regularly. Are there strong objections from the group?

**Comment by walshbr** [comment]
I'd say keep both in the template. Would probably help people use this one.

@jerielizabeth - one thing I found that I added to the draft text here was a note about whether or not the authors wished to get an initial round of feedback by email from me before hopping straight to GitHub. So I modified the leading sentence of the first full paragraph as such:

"The authors have requested an initial round of editorial feedback from me by email before moving to GitHub. In order to promote speedy publication of this important topic, we have agreed to a submission date of no later than May 17, 2017. The author agrees to contact the editor in advance if they need to revise the deadline."

That allows it to better document the contract I've established with the authors.
